odoo/tools/sql.py

```diff
@@ -594,18 +594,19 @@ def add_index(cr, indexname, tablename, definition, *, unique: bool, comment='')
         definition = SQL(definition.replace('%', '%%'))
     else:
         definition = SQL(definition)
-    cr.execute(SQL(
-        "CREATE %sINDEX %s ON %s %s",
-        SQL("UNIQUE ") if unique else SQL(),
-        SQL.identifier(indexname),
-        SQL.identifier(tablename),
-        definition,
-    ))
-    if comment:
+    with cr.savepoint(flush=False):
         cr.execute(SQL(
-            "COMMENT ON INDEX %s IS %s",
-            SQL.identifier(indexname), comment,
+            "CREATE %sINDEX %s ON %s %s",
+            SQL("UNIQUE ") if unique else SQL(),
+            SQL.identifier(indexname),
+            SQL.identifier(tablename),
+            definition,
         ))
```

Comment:
I think we don't want to log SQL queries here...
```suggestion
        ),
        log_exceptions=False)
```